.vscode/launch.json

```diff
@@ -57,7 +57,8 @@
 				"${workspaceFolder}/packages/langium/lib/**/*.js",
 				"${workspaceFolder}/examples/arithmetics/out/**/*.js",
 				"${workspaceFolder}/examples/domainmodel/out/**/*.js",
-				"${workspaceFolder}/examples/statemachine/out/**/*.js"
+				"${workspaceFolder}/examples/statemachine/out/**/*.js",
+				"${workspaceFolder}/packages/langium-vscode/out/**/*.js"
```

Comment:
Question: I was wondering for a long time why our source maps for the langium-extension itself didn't work. Did this fix it?

Reply:
This fix it for me at least. After adding this `out` entry I was able to set breakpoints in langium package files e.g. the `langium-grammar-validator.ts`

Reply:
Yeah, that makes sense, thanks 👍